src/pyOpenMS/pxds/ChromatogramSettings.pxd

```diff
@@ -25,42 +25,42 @@ cdef extern from "<OpenMS/METADATA/ChromatogramSettings.h>" namespace "OpenMS":
         Product getProduct() nogil except + # wrap-doc:Access to the product ion
         void setProduct(Product p) nogil except + # wrap-doc:Set the product ion
 
-        String getNativeID() nogil except + # wrap-doc:returns the native identifier for the spectrum, used by the acquisition software.
-        void setNativeID(String native_id) nogil except + # wrap-doc:sets the native identifier for the spectrum, used by the acquisition software.
+        String getNativeID() nogil except + # wrap-doc:Returns the native identifier for the spectrum, used by the acquisition software.
+        void setNativeID(String native_id) nogil except + # wrap-doc:Sets the native identifier for the spectrum, used by the acquisition software.
 
         # returns the free-text comment
-        String getComment() nogil except +
+        String getComment() nogil except + # wrap-doc:Returns the free-text comment
         # sets the free-text comment
-        void setComment(String comment) nogil except +
+        void setComment(String comment) nogil except + # wrap-doc:Sets the free-text comment
 
         # returns a mutable reference to the instrument settings of the current spectrum
-        InstrumentSettings getInstrumentSettings() nogil except +
+        InstrumentSettings getInstrumentSettings() nogil except + # wrap-doc:Returns a mutable reference to the instrument settings of the current spectrum
         # sets the instrument settings of the current spectrum
-        void setInstrumentSettings(InstrumentSettings instrument_settings) nogil except +
+        void setInstrumentSettings(InstrumentSettings instrument_settings) nogil except + # wrap-doc:Sets the instrument settings of the current spectrum
 
         # returns a mutable reference to the acquisition info
-        AcquisitionInfo getAcquisitionInfo() nogil except +
+        AcquisitionInfo getAcquisitionInfo() nogil except + # wrap-doc:Returns a mutable reference to the acquisition info
         # sets the acquisition info
-        void setAcquisitionInfo(AcquisitionInfo acquisition_info) nogil except +
+        void setAcquisitionInfo(AcquisitionInfo acquisition_info) nogil except + # wrap-doc:Sets the acquisition info
 
         # returns a mutable reference to the source file
-        SourceFile getSourceFile() nogil except +
+        SourceFile getSourceFile() nogil except + # wrap-doc:Returns a mutable reference to the source file
```

Comment:
Same

Reply:
And all following

Reply:
ok. I will verify the mutable methods in my local.